com.unity.render-pipelines.high-definition/Runtime/Material/SharedRTManager.cs

```diff
@@ -495,14 +524,22 @@ public void ResolveMSAAColor(CommandBuffer cmd, HDCamera hdCamera, RTHandle msaa
                 Debug.Assert(m_MSAASupported);
                 using (new ProfilingScope(cmd, ProfilingSampler.Get(HDProfileId.ResolveMSAAColor)))
                 {
-                    // Grab the RTIs and set the output render targets
-                    CoreUtils.SetRenderTarget(cmd, simpleTarget);
+                    if (DeviceInfo.requiresExplicitMSAAResolve == false &&
+                        CompareRTHandle(msaaTarget, simpleTarget))
+                    {
+                        cmd.ResolveAntiAliasedSurface(msaaTarget.rt, simpleTarget.rt);
```

Comment:
Same issue as above, our color resolve rely on a tonemap operation for the average, it is not just an average. We can't rely on auto resolve.